contrib/README.md

```diff
@@ -294,9 +294,11 @@ As long as either the native `Promise` is there, or that global has been spec-co
 
 ### `errfcb` Plugin
 
-`errfcb` plugin provides `errfcb()` on the main sequence instance API. Calling `errfcb()` returns an "error-first" style (aka "node-style") callback that can be used with any function that expects such a callback.
+The `errfcb` plugin provides `errfcb()` on the main sequence instance API. Calling `errfcb()` returns an "error-first" style (aka "node-style") callback that can be used with any function that expects such a callback.  Calling `errfcb()` also stops the sequence from proceeding until the "error-first" callback is invoked.
 
```

Comment:
I see what you're trying to do with this added sentence, and why that helps with clarity. But I'd actually recommend this paragraph be edited for even more clarity, so it reads like this:

----

The `errfcb` plugin provides `errfcb()` on the main sequence instance API. Calling `errfcb()` creates a step in the sequence that will wait to proceed, and returns an "error-first" style (aka "node-style") callback -- suitable for any utility that expects such a callback -- to signal this waiting sequence step.

Reply:
Done, and thanks for the quick response!  I pulled the "suitable for" out to a separate sentence just to break it up a bit.